docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md

```diff
@@ -3,4 +3,35 @@ name: vercel
 routeTemplate: ./data/component-templates/article.yml
 title: Deploying to Vercel
 ---
-# Walkthrough: Deploying to Vercel
\ No newline at end of file
+# Walkthrough: Deploying to Vercel
+
+
+Vercel is an all-in-one platform with Global CDN supporting static & Jamstack deployment. Vercel has first-class support for Next.js.
+
+Using Vercel, you can easily deploy the sample app to production. 
+
+We assume you have set up the Next.js sample application to connect to a Sitecore instance for the following steps.
+
+To deploy the Next.js sample app to production: 
+1. Execute the steps provided in the Next.js  guide for [Getting Started](https://nextjs.org/docs/deployment#getting-started) with deployment.
+2. In the root directory of the application, in the `.env` file, add or modify:
```

Comment:
In the case that these are different for dev/prod, would these be defined in Vercel?

Reply:
Dupe of Adam's comment.